### ISSUE TITLE
update-overlay: Command not found.

### ISSUE THREAD

**Issue by elyobelyob** [tprelog/iocage-homeassistant#60]
root@HomAssistant:/home/homeassistant/homeassistant # update-overlay
update-overlay: Command not found.

**Comment by elyobelyob** [comment]
Maybe I also need to rename from HomAssitant too :)

**Comment by elyobelyob** [comment]
But thanks for the effort for new Python... it took 30 mins building wheels, and 30 mins rebuilding HA

**Comment by tprelog** [comment]
> root@HomAssistant:/home/homeassistant/homeassistant 
> update-overlay: Command not found.


Hi @elyobelyob --  At this time, I've only applied the update to the master branch. On TrueNAS 12 you're likely still pulling from the 12.2-RELEASE branch. However, if you're currently on version `v7-2202.7.3` you can pull the overlay from master with the following command. This will download and extract the overlay the to `/` (root) of your plugin.

```
iocage console JAIL_NAME
# Press '0' to exit the menu
curl -sL https://github.com/tprelog/iocage-homeassistant/archive/master.tar.gz | tar -xzf - -C / --strip=2 -- "*iocage-homeassistant*/overlay/"
```

After that, log out and back in again and then the `update-overlay` command should work.


> Maybe I also need to rename from HomAssitant too :)

I really hope not -- That would imply your plugin is at least two years old and has never been updated

**Comment by tprelog** [comment]
> At this time, I've only applied the update to the master branch. On TrueNAS 12 you're likely still pulling from the 12.2-RELEASE branch.

Ok, I've merged the update back to the 12.2-RELEASE now.
